x-pack/plugins/security_solution/public/detections/components/alerts_table/timeline_actions/use_add_exception_flyout.tsx

```diff
@@ -32,20 +33,26 @@ export const useExceptionFlyout = ({
   onRuleChange,
   isActiveTimelines,
 }: UseExceptionFlyoutProps): UseExceptionFlyout => {
+  const { setAllTourStepsHidden } = useTourContext();
   const [openAddExceptionFlyout, setOpenAddExceptionFlyout] = useState<boolean>(false);
   const [exceptionFlyoutType, setExceptionFlyoutType] = useState<ExceptionListTypeEnum | null>(
     null
   );
 
-  const onAddExceptionTypeClick = useCallback((exceptionListType?: ExceptionListTypeEnum): void => {
-    setExceptionFlyoutType(exceptionListType ?? null);
-    setOpenAddExceptionFlyout(true);
-  }, []);
+  const onAddExceptionTypeClick = useCallback(
+    (exceptionListType?: ExceptionListTypeEnum): void => {
+      setExceptionFlyoutType(exceptionListType ?? null);
+      setAllTourStepsHidden(true);
```

Comment:
Hide the tour when flyout open to avoid this situation:

<img width="2549" alt="Screenshot 2024-05-02 at 09 29 50" src="https://github.com/elastic/kibana/assets/6295984/1b27cd8a-6bde-40b8-b295-213f6ceef518">